flags/4x3/my.svg

```diff
@@ -1,7 +1,26 @@
-<svg xmlns="http://www.w3.org/2000/svg" id="flag-icons-my" viewBox="0 0 640 480">
```

Comment:
Similarly, the viewBox for 4x3 flags should be 0 0 640 480. The flags all follow an identical viewBox and future flags/edits must adhere to this. Also, the id must be retained. Essentially, don't change the first line when not needed.